04-Multi-Band-Raster.Rmd

```diff
@@ -241,17 +251,12 @@ RGB_stack <- stack("NEON_RemoteSensing/HARV/HARV_RGB_Ortho.tif")
 #view attributes of stack object
 RGB_stack 
 
-#What is the CRS of the raster stack
-crs(RGB_stack)
-
-#What is the extent of the raster stack
-extent(RGB_stack)
-
 ```
 
-We can use `RGB_stack@layers` to view the attributes of each band in the raster
-object in r. We can also use the `plot` and `hist` functions on the raster stack
-to plot and view histograms of each band in the stack.
+To further explore the rasters, we can use `RGB_stack@layers` to get a 
```

Comment:
How about:
We can use `RGB_stack@layers` to view a summary attributes for each band in the raster stack.